### PR TITLE
"Next" button in docs

### LINKED ISSUE #4420: Add a "next" button option for docs

### PR DESCRIPTION
**Pull request by johnnyaug**
Use the page properties to create a "Next" button at the bottom

image

image

Resolves #4420.

### CODE REVIEW

File: docs/setup/storage/s3.md [hunk 1]
```diff
@@ -40,7 +29,7 @@ or the [S3 gateway](../../integrations/spark.md#access-lakefs-using-the-s3a-gate
             "s3:ListMultipartUploadParts"
          ],
          "Effect": "Allow",
-         "Resource": ["arn:aws:s3:::<STORAGE_NAMESPACE>/*"],
+         "Resource": ["arn:aws:s3:::<BUCKET_NAME>/*"],
```
Comment (Jonathan-Rosenberg) [inline docs/setup/storage/s3.md:32]: This should be `STORAGE_NAMESPACE`. This is to limit the actions to work on the exact lakeFS repository location

File: docs/setup/storage/s3.md [hunk 2]
```diff
@@ -6,29 +6,18 @@ parent: Prepare Your Storage
...
-Pick the one that best describes your use case and to add it, go to the `Permissions` tab, and paste it.
-
-## Permission Policy
-
-By setting this bucket policy you'll be able to perform metadata operations as well as to upload/download objects to/from your bucket using lakeFS.
-To upload or download an object to your storage namespace using lakeFS, you can use the [API](../../reference/api.md), the CLI tool
-[`lakectl`](../../reference#lakectl-fs), the [Spark metadata client](../../reference/spark-client.md),
-or the [S3 gateway](../../integrations/spark.md#access-lakefs-using-the-s3a-gateway).
+1. From the S3 Administration console, choose _Create Bucket_.
+2. Make sure that you block public access and disable object lock.
```
Comment (Jonathan-Rosenberg) [inline docs/setup/storage/s3.md:15]: I think that the removed links will be helpful here

File: docs/setup/storage/s3.md (hunk 1 above)
Comment (johnnyaug) [reply docs/setup/storage/s3.md:32]: To use @kesarwam's "crawl - walk - run" analogy, this page is IMO still in the "crawl" part. We want to get the user going as fast as possible. At this point, we're not sure they have already grasped the "storage namespace" term.

**Review by Jonathan-Rosenberg — CHANGES_REQUESTED** [review]
Great!
One comment in the `setup/storage/s3` page.

File: docs/setup/storage/s3.md (hunk 1 above)
Comment (Jonathan-Rosenberg) [reply docs/setup/storage/s3.md:32]: Ok.
To minimize issues and questions from users I would add a comment such as:

> To minimize lakeFS access to your bucket, replace `<BUCKET_NAME>` with your lakeFS repository complete [storage namespace](../../understand/model.md#concepts-unique-to-lakefs)

**Review by Jonathan-Rosenberg — APPROVED** [review]

**Pull request merged**